.github/workflows/dev-deploy-cloud-functions.yml

```
@@ -0,0 +1,78 @@
+      - '.github/workflows/*-deploy-cloud-functions.yml'
+
+env:
+  PROJECT_ID: ${{ secrets.PROJECT_ID }}
+  REGION: us-west1
+  ENVIRONMENT: dev
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)